### PR TITLE
Improve error handling

### PR DESCRIPTION
**Pull request by brunelli**
This PR adds error handling for login (missing login file, incomplete login file and wrong login info) and empty borrowed list.

I reworked the login experience by:

- Adding a CLI to ask for login and password.

- Removing the `login.txt` file from git. Now it is created when the program is first run (if the user chooses to save the login info) and is not tracked by git.

- Detecting when login fails and return feedback to the user, making it possible to update the `login.txt` content.

Also, if the "Renovar Todos" button is unavailable, the script will print the message returned.

I'm opening this as a WIP because it's a big change and I'd like feedback about possible errors and things we could do better before merging.

### CODE REVIEW

**Comment by brunelli** [comment]
Selenium was leaving `chromedriver` processes running, because I forgot to add a `driver.quit()` in 
27f88ea3727cdd0204a194125187ff931ba49008. It was not needed before because closing the browser was an user action, which is now impossible (as there's no window visible). It's added in 7ee3e986ff994d5d86cc0da10565b18e441a74db.

File: MinervaBot.py [hunk 1]
```diff
@@ -28,40 +51,71 @@
...
+while True:
+    # Acha as entradas para o id e senha
+    idcpf = driver.find_element_by_name('bor_id')
+    password = driver.find_element_by_name('bor_verification')
 
-# Preenche as entradas com os parametros dados
-idcpf.send_keys(inputcpf)
-password.send_keys(inputpassword, Keys.RETURN)
+    # Preenche as entradas com os parametros dados
+    idcpf.clear()
```
Comment (mvinoba) [inline MinervaBot.py:72]: I get it should be a good idea to always clear the text entry element before inserting the values, but why clearing only `idcpf` and not `password`?

**Review by mvinoba — COMMENTED** [review]
Thanks for the PR, lots of good improvements! 
I'll see if I can repackage it to windows binary (.exe) so the current release don't get too behind.

File: MinervaBot.py (hunk 1 above)
Comment (brunelli) [reply MinervaBot.py:72]: When login fails the `idcpf` field is filled with the value input on previous page, but `password` remains empty.

Comment (mvinoba) [reply MinervaBot.py:72]: @brunelli oh, I see, that's great! Everything with the login improvements looks good to me, I'm merging the PR.

**Pull request merged**